plugins/CoreAdminHome/lang/en.json

```
@@ -24,6 +24,8 @@
         "ForBetaTestersOnly": "For beta testers only",
         "HttpTrackingApi": "HTTP Tracking API",
         "HttpTrackingApiDescription": "The %1$sHTTP Tracking API%2$s allows you to track anything. This may be useful if you are using a programming language for which no SDK exists yet. It may also be useful when you want to track devices or application in a special way.",
+        "SinglePageApplication": "Single-Page Application or Progressive Web App",
+        "SinglePageApplicationDescription": "If you are using a %1$sSingle Page/Progressive web %2$sapplication, this helps you tracking the correct data.",
```

Isn't the link a bit misplaced? Currently `Single Page/Progressive web` is linked. I would expect the link leads me to a page that explains what that is. Actually the `this` indicates that there are more info, but it isn't linked. Maybe that makes more sense:
```suggestion
        "SinglePageApplicationDescription": "If you are using a Single Page/Progressive web application, %1$sthis guide%2$s helps you tracking the correct data.",
```